src/FamilyHubs.Referral.Web/StartupExtensions.cs

```diff
@@ -48,6 +50,14 @@ public static void AddWebUiServices(this IServiceCollection services, IConfigura
         // Customise default API behaviour
         services.Configure<ApiBehaviorOptions>(options => options.SuppressModelStateInvalidFilter = true);
 
+        services.AddDistributedMemoryCache();
+        services.AddSession(options => {
+            options.IdleTimeout = TimeSpan.FromMinutes(configuration.GetValue<int>("SessionTimeOutMinutes"));
+        });
+
+        services.AddTransient<IRedisCache, RedisCache>();
```

Comment:
The recommended way of accessing Redis seems to be using StackExchange.Redis, for features such as multiplexed connections. Could we swap over to using it?

Reply:
services.AddTransient<IRedisCache, RedisCache>(); <-- Is a wrapper and that uses StackExchange.Redis have a look at the NuGet Package